db/db_local/backup.go

```diff
@@ -160,26 +194,72 @@ func needsBackup(ctx context.Context) (bool, string, error) {
 	return false, "", nil
 }
 
-// loadBackup loads the back up file into the database
-func loadBackup(ctx context.Context) error {
-	if !helpers.FileExists(getBackupLocation()) {
+// restoreBackup loads the back up file into the database
+func restoreBackup(ctx context.Context) error {
+	if !helpers.FileExists(databaseBackupFilePath()) {
 		// nothing to do here
 		return nil
 	}
 
 	// load the db status
-	info, err := GetState()
+	runningInfo, err := GetState()
 	if err != nil {
 		return err
 	}
-	if info == nil {
+	if runningInfo == nil {
 		return fmt.Errorf("steampipe service is not running")
 	}
 
+	// get the location of the other instance which was backed up
+	_, location, err := needsBackup(ctx)
```

Comment:
```
_, location, err := needsBackup(ctx)
```
that is a pretty counterintuitive function call

Reply:
changed the fn name